app/grandchallenge/reader_studies/forms.py

```diff
@@ -310,6 +337,52 @@ def interface_choices(self):
     def initial_interface(self):
         return self.interface_choices().first()
 
+    def widget_choices(self):
+        answer_type = self["answer_type"].value()
+        choices = [("", "Default")]
+        if answer_type:
+            try:
+                extra_options = ANSWER_TYPE_TO_QUESTION_WIDGET[answer_type]
+                choices.extend(
+                    (option.name, option.label) for option in extra_options
```

Comment:
Is this possible?

```suggestion
                    option.choice for option in extra_options
```

Reply:
No, the `choice` attribute (or anything similar) doesn't exist on the option...